src/api/v3/imprint.py

```diff
@@ -0,0 +1,55 @@
+"""
+imprint API endpoint
+"""
+from django.http import JsonResponse
+
+from cms.models import Region
+
+
+def transform_imprint(imprint_translation):
+    """
+    Function to create a JSON from a single imprint_translation object.
+
+    :param imprint_translation: single page translation object
+    :type imprint_translation: ~cms.models.pages.page_translation.PageTranslation
+
+    :return: return data necessary for API
+    :rtype: dict
+    """
+    return {
+        "id": imprint_translation.id,
+        "url": imprint_translation.permalink,
+        "title": imprint_translation.title,
+        "modified_gmt": imprint_translation.last_updated,
+        "excerpt": imprint_translation.text,
+        "content": imprint_translation.text,
+        "parent": None,
+        "available_languages": imprint_translation.available_languages,
+        "thumbnail": None,
```

Comment:
```suggestion
        "thumbnail": imprint_translation.page.icon,
```
The `ImprintPage` model also has an `icon` attribute. Should we use it here?

Reply:
yes